tests/results/test_aggregator.py

```diff
@@ -9,6 +10,7 @@
 import h5py
 import numpy
 import pytest
+from _pytest.fixtures import FixtureRequest
```

Comment:
```suggestion
from pytest import FixtureRequest
```

Reply:
*scream in despair* Why did you fail me auto import!